lib/bill_forward/entities/account.rb

```diff
@@ -7,6 +7,22 @@ module BillForward
   class Account < MutableEntity
   	@resource_path = BillForward::ResourcePath.new("accounts", "account")
 
+    class << self
+      def credit(id, query_object = {}, custom_client = nil)
```

Comment:
why `query_object`?
this has nothing to do with queries.
there is a danger that this implies "query params" in the URL.

maybe something more like `request_object`.